mackup/appsdb.py

```diff
@@ -94,7 +94,20 @@ def get_config_files():
         # Configure the config parser
         apps_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                                 APPS_DIR)
-        custom_apps_dir = os.path.join(os.environ['HOME'], CUSTOM_APPS_DIR)
+        xdg_config_home = os.environ.get('XDG_CONFIG_HOME')
+        if xdg_config_home:
```

Comment:
it's a lot of copy pasta. Can you add a util that you can use to look for a file in XDG_CONFIG_HOME if set, else in HOME? And use it everywhere?